Sources/Common/Util/Log.swift

```diff
@@ -98,3 +98,35 @@ public class ConsoleLogger: Logger {
     public func error(_ message: String) {}
     #endif
 }
+
+extension CioLogLevel {
+    static func getLogLevel(for value: Int) -> CioLogLevel {
```

Comment:
I know that in react native, we send `int` values of the log levels. Does Flutter send string values as in the next function? I think we can club these two together and maintain consistency across the wrappers i.e. either use strings or ints, because the two functions do the same thing. Thoughts?

Reply:
Good point, we can align it, just wanted to be wary of the changes we might have to do to the wrappers, could it be a breaking change? or is it something that can be handled internally?

Reply:
This can be handled by making a small change in `CioLogLevel` in the package and this won't require any change from a user's end. Let's make a ticket to make and track this change.

Reply:
Would it be OK for the SDK wrappers to pass in a string instead of an Int data type? 

I am thinking that an Int could cause errors by accident in the future. If, for example, the native SDKs add/remove/edit one of the log levels. By the native SDK making a change to the log levels like this, we would need to modify this function and could introduce a bug into the SDK wrappers by accident. It looks easy to change one of the Int `case` statements and not consider the impact you could be making. 

If we used strings, I wonder if we could better avoid introducing a bug? 

```suggestion
    static func getLogLevel(for value: String) -> CioLogLevel {
      switch value {
      case "error": return .error
      case "info": return .info      
```

Reply:
I am seeing there is already a function that uses a `String` parameter. 

If the SDK wrappers currently use `Int` as the public API of the SDK, I would like to suggest the idea of converting `Int` into `String` inside of the SDK wrapper itself. Keep backwards compatibility while not adding logic to the native SDK that we need to maintain for a specific SDK wrapper?

Reply:
Removing the int option, as only RN uses `int` and it's not customer facing so they can change it internally.